Microsoft.Toolkit.Uwp.UI.Controls.Layout/GridSplitter/GridSplitter.Options.cs

```diff
@@ -92,24 +50,14 @@ public partial class GridSplitter
                 nameof(CursorBehavior),
                 typeof(SplitterCursorBehavior),
                 typeof(GridSplitter),
-                new PropertyMetadata(SplitterCursorBehavior.ChangeOnSplitterHover, CursorBehaviorPropertyChanged));
-
-        /// <summary>
-        /// Gets or sets the visual content of this Grid Splitter
-        /// </summary>
-        public UIElement Element
-        {
-            get { return (UIElement)GetValue(ElementProperty); }
-            set { SetValue(ElementProperty, value); }
-        }
+                new PropertyMetadata(SplitterCursorBehavior.ChangeOnSplitterHover));
 
         /// <summary>
         /// Gets or sets whether the Splitter resizes the Columns, Rows, or Both.
         /// </summary>
-        public GridResizeDirection ResizeDirection
+        public new GridResizeDirection ResizeDirection
```

Comment:
Confused why this is overridden? If the `ResizeDirection` has different enums for `GridSplitter` vs. `ContentSizer` we should just make them similarly named properties on each class vs. trying to have them shared on the base class.